src/pages/list/list.ts

```diff
@@ -19,13 +27,21 @@ export class ListPage {
     'american-football', 'boat', 'bluetooth', 'build'];
 
     this.items = [];
-    for (let i = 1; i < 11; i++) {
+    for (let i = 1; i < 5; i++) {
```

Comment:
why change this? this is anyway unused.. isnt it?

Reply:
it is, I reduced it because it takes all screen